apps/ledger-live-desktop/src/renderer/components/OperationsList/Operation.tsx

```diff
@@ -49,69 +44,58 @@ type OwnProps = {
+    confirmationsNbForCurrencySelector(state, mainAccount),
+  );
+  const unit = useAccountUnit(mainAccount);
 
-  onOperationClick = () => {
-    const { account, parentAccount, onOperationClick, operation } = this.props;
+  const onClickOnOperation = () => {
     onOperationClick(operation, account, parentAccount);
   };
 
-  render() {
-    const {
-      account,
-      parentAccount,
-      t,
-      operation,
-      withAccount,
-      text,
-      withAddress,
-      confirmationsNb,
-      editable,
-    } = this.props;
-    const isOptimistic = operation.blockHeight === null;
-    const currency = getAccountCurrency(account);
-    const unit = getAccountUnit(account);
-    const mainAccount = getMainAccount(account, parentAccount);
-    const isConfirmed = isConfirmedOperation(operation, mainAccount, confirmationsNb);
-    return (
-      <OperationRow
-        className="operation-row"
-        isOptimistic={isOptimistic}
-        onClick={this.onOperationClick}
-      >
-        <ConfirmationCell
-          operation={operation}
-          parentAccount={parentAccount}
-          account={account}
-          t={t}
-          isConfirmed={isConfirmed}
-        />
-        <DateCell
-          family={mainAccount.currency.family}
-          text={text}
-          operation={operation}
-          editable={editable}
-          t={t}
-        />
-        {withAccount && <AccountCell accountName={getAccountName(account)} currency={currency} />}
-        {withAddress ? <AddressCell operation={operation} /> : <Box flex="1" />}
-        <AmountCell
-          operation={operation}
-          currency={currency}
-          unit={unit}
-          isConfirmed={isConfirmed}
-        />
-      </OperationRow>
-    );
-  }
+  const isOptimistic = operation.blockHeight === null;
+  const currency = getAccountCurrency(account);
+
+  const isConfirmed = isConfirmedOperation(operation, mainAccount, confirmationsNb);
+  return (
+    <OperationRow
+      className="operation-row"
+      isOptimistic={isOptimistic}
+      onClick={onClickOnOperation}
+    >
+      <ConfirmationCell
+        operation={operation}
+        parentAccount={parentAccount}
+        account={account}
+        t={t}
+        isConfirmed={isConfirmed}
+      />
+      <DateCell
+        family={mainAccount.currency.family}
+        text={text}
+        operation={operation}
+        editable={editable}
+        t={t}
+      />
+      {withAccount && <AccountCell accountName={getAccountName(account)} currency={currency} />}
+      {withAddress ? <AddressCell operation={operation} /> : <Box flex="1" />}
+      <AmountCell operation={operation} currency={currency} unit={unit} isConfirmed={isConfirmed} />
+    </OperationRow>
+  );
 }
-const ConnectedOperationComponent: React.ComponentType<OwnProps> =
-  connect(mapStateToProps)(OperationComponent);
+const ConnectedOperationComponent: React.ComponentType<OwnProps> = OperationComponent;
```

Comment:
to make it iso, can you React.memo the component? since it was before a PureComponent